### DISCUSSION TITLE
Guidance Needed for Running RCAIDE Code in Latest Develop Branch

### DISCUSSION THREAD

**Discussion by JunrryShan** [leadsgroup/RCAIDE_LEADS#126] [Q&A] [answered]
Dear RCAIDE Development Team,

First of all, I want to express my deepest respect for your incredible work on the RCAIDE project. Your research and contributions to the field have been truly inspiring, and I am eager to learn from the innovative methodologies and applications you've developed.

I have been exploring the code provided in the RESEARCH folder and am particularly interested in running the examples in 02_Rotor_Blade_Optimization and 03_Aircraft_Noise_Modeling. However, I have encountered some challenges while trying to use the latest develop branch of the RCAIDE repository.

The develop branch seems to differ significantly from the versions used in these examples, requiring numerous modifications to make the code compatible. Despite my efforts, I have faced difficulties in getting the examples to run smoothly.

Could you kindly provide some guidance or advice on how to adapt the 02_Rotor_Blade_Optimization and 03_Aircraft_Noise_Modeling examples to the latest develop branch? Alternatively, is there a specific version of the RCAIDE codebase that you would recommend using for these examples to avoid compatibility issues?

Your assistance and suggestions would be invaluable in helping me better understand and utilize the tools you've developed. Thank you in advance for your time and for sharing your expertise.

Best regards,
Chengjun Shan

**Comment by ssankalp26** [comment, marked as answer]
Hello Chengjun Shan,

We have been updating a lot of things in the background, I would like to redirect you to the master branch where we have a host of tutorials on how RCAIDE works with more tutorials coming out soon.
You can also check out the documentation at https://www.docs.rcaide.leadsresearchgroup.com for detailed installation instructions for the latest version of RCAIDE-LEADS. 

Regards,
Sai Sankalp
